Equinox/Globals.h

```diff
@@ -54,5 +56,68 @@ typedef unsigned int uint;
 #define RESIZABLE true
 #define TITLE "Equinox Engine Editor"
 
+inline void DrawBoundingBox(AABB& boundingBox)
```

Comment:
It's not so dramatic. But could be inside a module and could be used also from anywhere, and we would keep the globals class clean. We have and we will have more global methods, and we wouldn't fill this class with them. (In my opinion)